projects/novo-elements/src/elements/chips/Chips.ts

```diff
@@ -222,9 +226,9 @@ export class NovoChipsElement implements OnInit, ControlValueAccessor {
       if (noLabels.length > 0 && this.source && this.source.getLabels && typeof this.source.getLabels === 'function') {
         this.source.getLabels(noLabels).then((result) => {
           for (let value of result) {
-            if (value.hasOwnProperty('label')) {
+            if (value.hasOwnProperty('label') && value.hasOwnProperty('value')) {
               this.items.push({
```

Comment:
what is the value in the item list being used for? and what about the scenario when there is no `value.value`